jekyll/_cci2/configuration-reference.md

```diff
@@ -313,17 +314,17 @@ Template | Description
 {% raw %}`{{ epoch }}`{% endraw %} | the current time in seconds since the unix epoch.
 {: class="table table-striped"}
 
-During step execution template will be replaced by runtime values and resulted string will be cache key.
+During step execution, the templates above will be replaced by runtime values and using the resultant string as the `key`.
```

Comment:
"During step execution, the templates above will be replaced by runtime values and *use* the resultant string as the `key`."

What's the difference between 'resultant' and 'resulting'?